src/app/EventManagement.cpp

```diff
@@ -124,6 +124,16 @@ void EventManagement::Init(Messaging::ExchangeManager * apExchangeManager, uint3
     mBytesWritten = 0;
 
     mMonotonicStartupTime = aMonotonicStartupTime;
+
+    // Should remove using the global instance and rely only on passed in variable.
```

Comment:
So here's my question: this Init method is only called in one place, in Server, right?  Why do we need to keep backwards compat here?  This is not public API....

But it's probably OK to do that in a followup.  What needs to happen then is:

1. The followup is filed.
2. The comment here points to that issue.
3. There is a commitment to fix the issue.

Reply:
Added follow up issue and will fix it as a follow up